styles.css

```diff
@@ -0,0 +1,8 @@
+@font-face {
```

Comment:
Same with my previous comment, it would be best to define these styles in the overrides where possible.